gnome-initial-setup/demo/eos-demo-video.c

```diff
@@ -132,14 +137,32 @@ gis_demo_video_init (GisDemoVideo *demo_video)
   GisDemoVideoPrivate *priv = gis_demo_video_get_instance_private (demo_video);
   GstBus *bus = NULL;
 
+  const gchar * const *languages;
+  gchar file_path[PATH_MAX];
```

Comment:
Where is `PATH_MAX` defined?

Reply:
It was included with something else that I included, but probably best to explicitly include <limits.h>.  Also, I realized that I added <stdio.h> from my test code, but that should not really be needed and I'll remove it.